dev-lang/ghc/ghc-8.4.3.ebuild

```diff
@@ -67,7 +67,7 @@ BUMP_LIBRARIES=(
 
 LICENSE="BSD"
 SLOT="0/${PV}"
-KEYWORDS="~alpha ~amd64 ~x86 ~amd64-linux ~x86-linux"
+KEYWORDS="-alpha ~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
"-alpha" means "will likely never work on alpha". It should work just fine. Just drop `~alpha` from the list.

Reply:
No worries. I'll get this fixed in about 24 hours.